src/app/routes/connector-ui/asset-page/asset-edit-dialog/asset-edit-dialog.component.html

```diff
@@ -180,6 +180,181 @@ <h1 mat-dialog-title>
+            </mat-form-field>
+
+            <!-- Geo location -->
+            <mat-form-field class="grow">
+              <mat-label>Geo location</mat-label>
+              <input
+                matInput
+                [formControl]="form.advanced.controls.geoLocation" />
+            </mat-form-field>
+          </div>
+
+          <!-- NUTS locations -->
+          <div class="form-section-title">NUTS locations</div>
+          <div
+            *ngFor="
+              let location of form.advanced.controls.nutsLocations.controls;
+              let i = index
+            "
+            class="flex flex-row space-x-[10px]">
+            <mat-form-field class="grow">
+              <mat-label>NUTS location</mat-label>
+              <input matInput placeholder="DE929" [formControl]="location" />
+            </mat-form-field>
+
+            <button
+              mat-button
+              color="warn"
+              style="height: 54px; margin-top: 4px; margin-left: 8px"
+              (click)="form.onNutsLocationsRemoveClick(i)">
+              Remove
+            </button>
+          </div>
+
+          <div class="flex flex-row mb-[10px]">
+            <button
+              mat-button
+              color="accent"
+              (click)="form.onNutsLocationsAddClick()">
+              Add location
+            </button>
+          </div>
+
+          <!-- Data sample URLs -->
+          <div class="form-section-title">Data samples</div>
+          <div
+            *ngFor="
+              let sample of form.advanced.controls.dataSampleUrls.controls;
+              let i = index
+            "
+            class="flex flex-row space-x-[10px]">
+            <mat-form-field class="grow">
+              <mat-label>URL</mat-label>
+              <input
+                matInput
+                placeholder="https://example.com"
+                [formControl]="sample" />
+            </mat-form-field>
+
+            <button
+              mat-button
+              color="warn"
+              style="height: 54px; margin-top: 4px; margin-left: 8px"
+              (click)="form.onDataSampleUrlsRemoveClick(i)">
+              Remove
+            </button>
+          </div>
+
+          <div class="flex flex-row mb-[10px]">
+            <button
+              mat-button
+              color="accent"
+              (click)="form.onDataSampleUrlsAddClick()">
+              Add data sample
+            </button>
+          </div>
+
+          <!-- Reference file URLs -->
+          <div class="form-section-title">Reference files</div>
+          <div
+            *ngFor="
+              let file of form.advanced.controls.referenceFileUrls.controls;
+              let i = index
+            "
+            class="flex flex-row space-x-[10px]">
+            <mat-form-field class="grow">
+              <mat-label>URL</mat-label>
+              <input
+                matInput
+                placeholder="https://example.com"
```

Comment:
we could maybe adjust this to `https://my-org.com/my-data-offer/documentation/api-reference`